src/main/java/AppScreen.java

```diff
@@ -0,0 +1,119 @@
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+
+
+public class AppScreen implements AppScreenPresenter, AppScreenController {
+
+    final JFrame JFRAME;
+    final User CURR_USER;
+    public ArrayList<Chat> chats;
+
+
+    /*
```

Comment:
This is a detail, but if you want a Javadoc, it begins with /** (two asterisks).
e.g.
```
/**
* This is a class, it does X, Y, Z
*/
```
It's good, however, that you have documented your methods.

Reply:
Ah I see, sorry I haven't used javadoc in about 2 years so my knowledge on it is a bit rusty